app/views/CinematicCanvas.vue

```diff
@@ -0,0 +1,45 @@
+<style scoped>
+#cinematic-div {
+  position: relative;
+  display: inline-block;
+  font-size: 1.5em;
+}
+
+#cinematic-div canvas {
+  display: block;
+  position: absolute;
+}
+</style>
+
+<template>
+  <div :height="this.height" id="cinematic-div" ref="cinematic-div">
+    <canvas :width="this.width" :height="this.height" id="cinematic-canvas" ref="cinematic-canvas"></canvas>
+  </div>
+</template>
+
+<script>
+/**
+ * This vue component initializes the cinematic experience via the
+ * CinematicController.
+ */
+import { CinematicController } from './play/cinematic/cinematicController'
+
+export default {
+  props: {
+    slug: {
+      type: String,
+      required: true
+    }
+  },
+  data: () => ({
+    controller: null,
+    width: 800,
+    height: 514,
```

Comment:
Why are we defining height and width as data and not in the `style` section?

Reply:
Good pickup. This was originally going to be a prop that can be passed in.
I will delete this for now and add it directly to the canvas element.